.cfnlintrc.yaml

```diff
@@ -47,6 +47,7 @@ ignore_templates:
   - tests/translator/output/**/connector_bucket_to_function.json
   - tests/translator/output/**/connector_dependson_replace.json
   - tests/translator/output/**/connector_esm_dependson.json
+  - tests/translator/output/**/connector_function_to_location.json
```

Comment:
What does `cfn-lint` complain about? If there's an easy way to keep it happy, that would be preferred over ignoring the file outright.

Reply:
Nothing after running `make lint` so have removed the ignore